File: myframework/src/main/java/me/kzv/mvc/web/controller/HomeController.java
package me.kzv.mvc.web.controller;


import me.kzv.mvc.annotation.MyController;
import me.kzv.mvc.annotation.MyRequestMapping;
import me.kzv.mvc.annotation.MyRequestMethod;
import me.kzv.mvc.web.repository.UserRepository;

import javax.servlet.http.HttpServletRequest;
import javax.servlet.http.HttpServletResponse;

@MyController
public class HomeController {
    @MyRequestMapping(value = "/", method = MyRequestMethod.GET)
    public String handleRequest(HttpServletRequest request, HttpServletResponse response) {
        request.setAttribute("users", UserRepository.findAll());
        return "home";
    }
}

File: JPA/kotlin-jpa-querydsl/src/main/kotlin/me/kzv/kotlinjpaquerydsl/controller/HomeController.kt
package me.kzv.kotlinjpaquerydsl.controller

import me.kzv.kotlinjpaquerydsl.utils.logger
import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping

@Controller
class HomeController {

    @GetMapping("/")
    fun home(): String{
        logger().info("home controller")

        return "home"
    }

}